packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -405,6 +405,41 @@ export const Table = <
     return flatColumns.some((column) => column.id === SELECTION_CELL_ID);
   }, [columns]);
 
+  const disableUserSelect = React.useCallback(
+    (e: KeyboardEvent) => {
+      if (e.key === 'Shift') {
+        ownerDocument &&
+          (ownerDocument.documentElement.style.userSelect = 'none');
+      }
+    },
+    [ownerDocument],
+  );
+
+  const enableUserSelect = React.useCallback(
+    (e: KeyboardEvent) => {
+      if (e.key === 'Shift') {
+        ownerDocument && (ownerDocument.documentElement.style.userSelect = '');
+      }
+    },
+    [ownerDocument],
+  );
+
+  React.useEffect(() => {
+    // Add onkeydown event listeners (during creation)
+    if (selectionMode === 'multi') {
+      document.addEventListener('keydown', disableUserSelect);
+      document.addEventListener('keyup', enableUserSelect);
+    }
+
+    // Remove onkeydown event listeners (during destruction)
```

Comment:
These comments are not necessary because that's just how useEffect works

Reply:
**[DONE]** Removed